workspaces/src/network/betanet.rs

```diff
@@ -21,7 +21,7 @@ pub struct Betanet {
 }
 
 impl Betanet {
-    pub(crate) async fn new() -> anyhow::Result<Self> {
+    pub(crate) async fn new() -> crate::result::Result<Self> {
```

Comment:
```suggestion
    pub(crate) async fn new() -> crate::Result<Self> {
```

I think we  should re-export result/error at root, so that call-site could look like `workspaces::Result`